test/integration/connection-monitoring-and-pooling/connection_monitoring_and_pooling.test.ts

```diff
@@ -23,4 +27,37 @@ describe('Connection Monitoring and Pooling (Node Driver)', function () {
     '../integration/connection-monitoring-and-pooling/unified-cmap-node-specs'
   );
   runUnifiedSuite(unifiedTests);
+
+  describe('ConnectionPoolCreatedEvent', () => {
+    let client: MongoClient;
+    beforeEach(async function () {
+      client = this.configuration.newClient();
+    });
+
+    afterEach(async function () {
+      await client.close();
+    });
+
+    describe('constructor()', () => {
+      it('when auth is enabled redacts credentials from options', {
+        metadata: { requires: { auth: 'enabled' } },
+        async test() {
+          const poolCreated = once(client, 'connectionPoolCreated');
+          await client.connect();
+          const [event] = await poolCreated;
```

Comment:
same comment as on the 4.x branch

Reply:
Same assertion, note the small difference in where the pool property is on the Server